model/waterfall.go

```diff
@@ -10,9 +10,11 @@ import (
 	"github.com/mongodb/anser/bsonutil"
 	"github.com/pkg/errors"
 	"go.mongodb.org/mongo-driver/bson"
+	"go.opentelemetry.io/otel/attribute"
 )
 
 const (
+	buildsKey                    = "builds"
```

Comment:
nit: since this is only used as a private const in getVersionTasksPipeline, we can just define it as a const in there

Reply:
It's also used by GetWaterfallBuildVariants